src/pages/slots/SlotConfiguration.styles.ts

```diff
@@ -112,21 +125,171 @@ export const EditCapacity = styled.button`
         margin-top: 4px;
     }
 `
-export const Title = styled.h1`
-    font-size: 24px;
-    margin-right:80px;
+export const SlotsContainer = styled.div`
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 520px;
+    min-height: 336px;
+    border: 2px solid ${TERTIARY_COLOR};
+    border-radius: ${BORDER_RADIUS};
+    gap:8px;
+    max-height: 512px; 
+    padding-bottom:4px;
+    overflow-y: auto; 
+`;
+export const TitlesContainer = styled.div`
+    display: flex;
+    flex-direction: column;
+    margin-right:300px;
+    gap:2px;
+    font-family: ${FONT_FAMILY};
+`;
+
+export const MainTitle = styled.h1`
+    font-size: 16px;
+    font-weight: bold;
+    margin:0px;
     margin-top:32px;
+    font-weight: 700; 
+`;
+
+export const Subtitle = styled.h2`
+    font-size: 12px;
+    margin:0px;
+    font-weight: 500;
+`;
+
+interface SpecificSlotContainerProps {
+    $isLast?: boolean;
+}
+
+export const SpecificSlotContainer = styled.div<SpecificSlotContainerProps>`
+    display: flex;
+    flex-direction: row;
+    width: 456px;
+    border-bottom: ${(props) => (props.$isLast ? "none" : `1px solid ${TEXT_COLOR}`)};
+    margin-bottom: ${(props) => (props.$isLast ? "px" : "none")};
+    padding:2px;
+    img{
+        margin-top:10px;
+        margin-right:16px;
+    } 
+       &:hover{
+        transform: scale(1.02);
+        transition: all 0.2s ease;
+    }
+         &.highlight {
+        border: solid 2px ${SUCCESS_COLOR};
+        background-color: ${SUCCESS_COLOR};
+        animation: highlightEffect 2s ease-in-out;
+    }
+    
+    @keyframes highlightEffect {
+        0% { 
+            background-color: transparent;
+            border-radius: ${BORDER_RADIUS};
+             border: solid 2px ${SUCCESS_COLOR};
+        }
+        10% { 
+            background-color: ${SUCCESS_COLOR}25;
+            border-radius: ${BORDER_RADIUS};
+             border: solid 2px ${SUCCESS_COLOR}80;
+           
+        }
+        20% { 
+            background-color: ${SUCCESS_COLOR}20;
+            border-radius: ${BORDER_RADIUS};
+                border: solid 2px ${SUCCESS_COLOR}75;
+        }
+        30% { 
+            background-color: ${SUCCESS_COLOR}25;
+            border-radius: ${BORDER_RADIUS};
+            border: solid 2px ${SUCCESS_COLOR}80;
+            
+        }
+        40% { 
+            background-color: ${SUCCESS_COLOR}20;
+            border-radius: ${BORDER_RADIUS};
+                border: solid 2px ${SUCCESS_COLOR}75;
```

Comment:
En esta parte no es necesario usar tantos porcentajes, se podría simplificar un poco teniendo el inicio, medio y fin y se logra el mismo efecto (o me parece que es igual):

```suggestion
@keyframes highlightEffect {
    0% {
      background-color: transparent;
      border-radius: ${BORDER_RADIUS};
      border: solid 2px ${SUCCESS_COLOR};
    }
    50% {
      background-color: ${SUCCESS_COLOR}30;
      border-radius: ${BORDER_RADIUS};
      border: solid 2px ${SUCCESS_COLOR}40;
    }
    100% {
      background-color: transparent;
      border: solid 2px transparent;
    }
  }
`;



```